### PR TITLE
Update error status of Secret Management API

### PR DESCRIPTION
**Pull request by ChanikaRuchini**
**Purpose**
$subject
> Related issue - [#12264](https://github.com/wso2/product-is/issues/12264)

### CODE REVIEW

File: components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java [hunk 1]
```diff
@@ -72,7 +77,7 @@
 
         public String getCode() {
 
-            return ERROR_PREFIX + code;
```
Comment (senthalan) [inline components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java:75]: As we change this, don't we need to add prefix to the existing codes

Comment (ChanikaRuchini) [reply components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java:75]: It already has a prefix as SECRETM. mistakenly added twice before

Comment (ChanikaRuchini) [reply components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java]: resolved

File: components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java [hunk 2]
```diff
@@ -36,14 +35,19 @@
     public enum ErrorMessage {
 
         // Client errors 600xx.
+        ERROR_CODE_CONFLICT_SECRET("00006", "Secret already exists.", "There exists a " +
```
Comment (senthalan) [inline components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java:38]: won't these error codes need to be started with 600 ?

And can order them in the error code order for better readability?

Comment (ChanikaRuchini) [reply components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java:38]: This error is getting from the framework.
 https://github.com/wso2/carbon-identity-framework/blob/4745401388cab18c7513a5802ed64670abf1b2b3/components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/constant/SecretConstants.java#L45

Comment (ChanikaRuchini) [reply components/org.wso2.carbon.identity.api.server.secret.management/org.wso2.carbon.identity.api.server.secret.management.common/src/main/java/org/wso2/carbon/identity/api/server/secret/management/common/SecretManagementConstants.java]: resolved. reused the code defined in framework

**Review by senthalan — APPROVED** [review]

**Pull request merged**